src/pages/App.tsx

```diff
@@ -25,7 +25,7 @@ import { Slide, ToastContainer } from 'react-toastify'
 import 'react-toastify/dist/ReactToastify.css'
 import Bridge from './Bridge'
 
-import Rewards from './Rewards'
```

Comment:
@adamazad I think for consistency it would be good to make all pages default export...And not make exception for rewards